tests/mocha/layering_test.js

```diff
@@ -80,7 +89,7 @@ suite('Layering', function () {
   });
 
   suite('dragging', function () {
-    test('moving an element to the drag layer adds it to the drag group', function () {
+    test.only('moving an element to the drag layer adds it to the drag group', function () {
```

Comment:
Reset to `test(` so you're not skipping all the other tests.

Reply:
I forget this far too often until I do my self-review. :) Thanks & removed in latest.